test/unit-test/FreeRTOS_DNS_Cache/FreeRTOS_DNS_Cache_utest.c

```diff
@@ -133,6 +146,87 @@ void test_processDNS_CACHE_Success( void )
     TEST_ASSERT_EQUAL( pulIP, x );
 }
 
+/**
+ * @brief Ensures that the same entry is inserted into the cache and retrieved for IPv6
+ */
+void test_processDNS_CACHE_Success2( void )
+{
+    uint32_t ulReturn = 0U;
+    uint32_t pulIP = 1234;
+    IPv46_Address_t pxIP;
+
+    pxIP.ulIPAddress = pulIP;
+    pxIP.xIs_IPv6 = pdTRUE;
+
+    xTaskGetTickCount_ExpectAndReturn( 3000 ); /* 3 seconds */
+    FreeRTOS_inet_ntop_ExpectAnyArgsAndReturn( NULL );
+
+    FreeRTOS_dns_update( "hello",
+                         &pxIP,
+                         FreeRTOS_htonl( 3 ), pdFALSE, NULL ); /* lives 3 seconds */
+
+    xTaskGetTickCount_ExpectAndReturn( 5000 );                 /* 5 seconds */
+    FreeRTOS_inet_ntop_ExpectAnyArgsAndReturn( NULL );
+
+    ulReturn = FreeRTOS_dnslookup6( "hello", &pxIP );
+
+    TEST_ASSERT_EQUAL( ulReturn, 1 );
+}
+
+/**
+ * @brief Ensures that failure occurs when different entry is inserted into the cache and retrieved for IPv6
+ */
+void test_processDNS_CACHE_Fail_IPv6( void )
+{
+    uint32_t ulReturn = 0U;
+    uint32_t pulIP = 1234;
+    IPv46_Address_t pxIP;
+
+    pxIP.ulIPAddress = pulIP;
+    pxIP.xIs_IPv6 = pdFALSE;
+
+    xTaskGetTickCount_ExpectAndReturn( 3000 ); /* 3 seconds */
+    FreeRTOS_inet_ntop_ExpectAnyArgsAndReturn( NULL );
+
+    FreeRTOS_dns_update( "hello",
+                         &pxIP,
+                         FreeRTOS_htonl( 3 ), pdFALSE, NULL ); /* lives 3 seconds */
+
+    xTaskGetTickCount_ExpectAndReturn( 10000 );                /* 5 seconds */
```

Comment:
The comment and code doesn't seem to match. Isn't it 10 seconds? 
Similarly in line 513, 663, 694.